geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -678,21 +694,15 @@ private void recoverRedundancy(Set<ServerLocation> excludedServers, boolean reco
             if (recoverInterest && queueConnections.getPrimary() == null
                 && queueConnections.getBackups().isEmpty()) {
               // we lost our queue at some point. We Need to recover
-              // interest. This server will be made primary after this method
-              // finishes
-              // because whoever killed the primary when this method started
-              // should
+              // interest. This server will be made primary after this method finishes
+              // because whoever killed the primary when this method started should
               // have scheduled a task to recover the primary.
               isFirstNewConnection = true;
               // TODO - Actually, we need a better check than the above. There's
```

Comment:
If the TODO still relevant? Perhaps the comment is old and needs to just go away or more work needs to be done to finish this code.

Reply:
It is written in such a way, that I don't actually know even after looking at the code.

Reply:
Done